auth/store_test.go

```diff
@@ -230,6 +231,119 @@ func TestRoleAdd(t *testing.T) {
 	}
 }
 
+func TestUserOperations(t *testing.T) {
+	b, tPath := backend.NewDefaultTmpBackend()
+	defer func() {
+		b.Close()
+		os.Remove(tPath)
+	}()
+
+	as := NewAuthStore(b, dummyIndexWaiter)
+	defer as.Close()
+	roleAddErr := enableAuthAndCreateRoot(as)
+	if roleAddErr != nil {
+		t.Fatal(roleAddErr)
+	}
+
+	// adds a new role
+	_, roleAddErr = as.RoleAdd(&pb.AuthRoleAddRequest{Name: "role-test"})
+	if roleAddErr != nil {
+		t.Fatal(roleAddErr)
+	}
+	as.UserAdd(&pb.AuthUserAddRequest{Name: "user1", Password: "xyz"})
+	as.UserGrantRole(&pb.AuthUserGrantRoleRequest{User: "user1", Role: "role-test"})
+
+	t.Run("Test Get User", func(t *testing.T) {
```

Comment:
Thanks for enhancing the test but each test should be separated into individual functions e.g. `TestGetUser()`. Could you separate them?